malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java

```diff
@@ -0,0 +1,40 @@
+package com.peacecorps.malaria.services;
+
+import android.app.Service;
+import android.content.Intent;
+import android.media.Ringtone;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.IBinder;
+import android.util.Log;
+
+public class RingService extends Service {
+    private Ringtone ringtone;
+
+    public RingService() {
+    }
+
+    @Override
+    public IBinder onBind(Intent intent)
+    {
+        return null;
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId)
+    {
+        Uri ringtoneUri = Uri.parse(intent.getExtras().getString("ringtone"));
+        Log.e("RingService OnStart","Starting Song");
+        this.ringtone = RingtoneManager.getRingtone(this, ringtoneUri);
+        ringtone.play();
+
+        return START_NOT_STICKY;
+    }
+
+    @Override
+    public void onDestroy()
+    {
+        Log.e("RingService OnEnd", "Stopping Song");
+        ringtone.stop();
+    }
+}
```

Comment:
Further be consistent with the rest of the codebase, in the sense that place the curly braces on the same line.

Reply:
@anubhakushwaha ma'am, can you please tell me which line the curly braces are out of line?